ImageGallery/src/org/sleuthkit/autopsy/imagegallery/actions/OpenAction.java

```diff
@@ -172,7 +172,15 @@ public void performAction() {
 
             switch (answer) {
                 case JOptionPane.YES_OPTION:
-                    ImageGalleryController.getDefault().setListeningEnabled(true);
+                    
+                    if (currentCase.getCaseType() == Case.CaseType.SINGLE_USER_CASE) {
+                        // toggling listening to ON automatically triggers a rebuild
+                        ImageGalleryController.getDefault().setListeningEnabled(true);
+                    }
+                    else {
+                        ImageGalleryController.getDefault().rebuildDB();
+                    }
+                
```

Comment:
We should comment this to explain the two different behaviors, as discussed in conversation.

Reply:
Excellent comment!!